index.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const fs = require('fs');
+const util = require('util');
+
+const writeFile = util.promisify(fs.writeFile);
+
+// ----------------------------------------------------------------------
+// Assignment 1: Code that writes code ...
+
+// Creates a file called loop.js in the files folder with whatever passed into it
+function makeLoopJS(content) {
+  writeFile(`${__dirname}/files/loop.js`, content)
+    .then(() => {
+      console.log('Created loop.js in the files folder');
+    })
+    .catch(err => {
+      console.log(err);
+    });
+}
+
+// Content I want to pass into my writeFile function
+let loopJsContent = new Int8Array([39, 117, 115, 101, 32, 115, 116, 114, 105, 99, 116, 39, 59, 10, 10, 40, 40, 41, 32, 61, 62, 32, 123, 10, 32, 32, 108, 101, 116, 32, 110, 97, 109, 101, 115, 32, 61, 32, 91, 39, 66, 105, 108, 108, 121, 39, 44, 32, 39, 74, 111, 110, 39, 44, 32, 39, 67, 111, 114, 121, 39, 93, 59, 10, 32, 32, 110, 97, 109, 101, 115, 46, 102, 111, 114, 69, 97, 99, 104, 40, 110, 97, 109, 101, 32, 61, 62, 32, 99, 111, 110, 115, 111, 108, 101, 46, 108, 111, 103, 40, 110, 97, 109, 101, 41, 41, 59, 10, 125, 41, 40, 41, 59, 10]);
```

Comment:
Sorry, taking another look at this. Your loop should translate a string into a buffer and then write that buffer to a new file.